Lua/Plugins/cpp.lua

```diff
@@ -0,0 +1,188 @@
+highlight("module", "reserved")
+highlight("mutable", "reserved")
+highlight("namespace", "reserved")
+highlight("new", "reserved")
+highlight("noexcept", "reserved")
+highlight("not", "reserved")
+highlight("not_eq", "reserved")
+highlight("nullptr", "reserved")
+highlight("operator", "reserved")
+highlight("or", "reserved")
+highlight("or_eq", "reserved")
+highlight("private", "reserved")
+highlight("protected", "reserved")
+highlight("public", "reserved")
+highlight("reflexpr", "reserved")
+highlight("register", "reserved")
+highlight("reinterpret_cast", "reserved")
+highlight("requires", "reserved")
+highlight("return", "reserved")
+highlight("short", "reserved")
+highlight("signed", "reserved")
+highlight("sizeof", "reserved")
+highlight("static", "reserved")
+highlight("static_assert", "reserved")
+highlight("static_cast", "reserved")
+highlight("struct", "reserved")
+highlight("switch", "reserved")
+highlight("synchronized", "reserved")
+highlight("template", "reserved")
+highlight("this", "reserved")
+highlight("thread_local", "reserved")
+highlight("throw", "reserved")
+highlight("true", "reserved")
+highlight("try", "reserved")
+highlight("typedef", "reserved")
+highlight("typeid", "reserved")
+highlight("typename", "reserved")
+highlight("union", "reserved")
+highlight("unsigned", "reserved")
+highlight("using", "reserved")
+highlight("virtual", "reserved")
+highlight("void", "reserved")
+highlight("volatile", "reserved")
+highlight("wchar_t", "reserved")
+highlight("while", "reserved")
+highlight("xor", "reserved")
+highlight("xor_eq", "reserved")
+
+--- Arithmetic Operators
+highlight("+", "operator")
+highlight("-", "operator")
+highlight("*", "operator")
+highlight("/", "operator")
+highlight("%", "operator")
+highlight("**", "operator")
+highlight("++", "operator")
+highlight("--", "operator")
+
+--- Assignment Operators
+highlight("=", "operator")
+highlight("+=", "operator")
+highlight("-=", "operator")
+highlight("*=", "operator")
+highlight("/=", "operator")
+highlight("%=", "operator")
+
+--- Comparison Operators
+highlight("==", "operator")
+highlight("!=", "operator")
+highlight(">", "operator")
+highlight("<", "operator")
+highlight(">=", "operator")
+highlight("<=", "operator")
+
+--- Logical Operators
+highlight("&&", "operator")
+highlight("||", "operator")
+highlight("!", "operator")
+
+--- Bitwise Operators
+highlight("&", "operator")
+highlight("|", "operator")
+highlight("^", "operator")
+highlight("~", "operator")
+highlight("<<", "operator")
+highlight(">>", "operator")
+
+--- Special Characters
+highlight("{", "binary")
+highlight("}", "binary")
+highlight("[", "binary")
+highlight("]", "binary")
+highlight("(", "binary")
+highlight(")", "binary")
+highlight(";", "binary")
+highlight(",", "binary")
+
+--- Strings
+highlight_region("\"", "\"", "string")
+highlight_region("'", "'", "string")
+
+--- Comments
+add_comment("//", "", "comments", true)
+add_comment("/*", "*/", "comments", true)
```

Comment:
`/* */` should not be line_only. pls set it to false or leave it as default

Reply:
Done, thank you